### PR TITLE
Minor improvements in localvir.md

### LINKED ISSUE #1996: Minor improvements in README.md
**Description :**
There are many deprecated dependencies due to which many users are facing issues while installing badgeyay frontend locally using npm install, so we can add a tip for such users to install through yarn.
We can also add a postgres installation command in the backend installation guide.

**Screenshots**
image

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by creyente1897**
Fixes #1996 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.

#### Changes proposed in this pull request:

* Added postgresql installation command

### CODE REVIEW

File: api/docs/installation/localvir.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ These are some additional depencies that you will need:
 $ sudo apt-get update
 ```
 
+* Postgres
```
Comment (rajvaibhavdubey) [inline api/docs/installation/localvir.md]: Add this before step 3 and also change the heading to Postgres Installation (also make it bold) . :smile:

Comment (creyente1897) [reply api/docs/installation/localvir.md]: ok sir :+1:

Comment (creyente1897) [reply api/docs/installation/localvir.md]: > Add this before step 3 
Sir can you please explain what to add?

Comment (creyente1897) [reply api/docs/installation/localvir.md]: Ok sir will do it  :+1:

**Comment by creyente1897** [comment]
@abishekvashok  Sir can you please review

**Pull request merged**